### ISSUE TITLE
Included ".s" file breakpoint is not triggered

### ISSUE THREAD

**Issue by prb28** [prb28/vscode-amiga-assembly#90]
How to reproduce:
Use the example workspace project: https://github.com/prb28/vscode-amiga-wks-example
Add a test.s file:
```assembly
testout:
  moveq    #1,d0
  moveq    #2,d0
  moveq    #3,d0  
  moveq    #4,d0
``` 
modify gencop.s adding at line 18:
```assembly
  include     "test.s"
```

Set a breakpoint in test.s line 5.
The breakpoint is not triggered.

**Comment by prb28** [comment]
Vasm "-linedebug" option does not handle the include in linedebug.

**Comment by prb28** [comment]
Fixed in vasm 1.8g (not yet released).
